docs/sources/dashboards/share-dashboards-panels/index.md

```diff
@@ -63,93 +66,147 @@ The same permission is also required to view embedded links unless you have anon
 As of Grafana 8.0, anonymous access permission is not available in Grafana Cloud.
 {{< /admonition >}}
 
-When you share a panel or dashboard as a snapshot, a snapshot (which is a panel or dashboard at the moment you take the snapshot) is publicly available on the web. Anyone with a link to it can access it. Because snapshots do not require any authorization to view, Grafana removes information related to the account it came from, as well as any sensitive data from the snapshot.
-
 ## Share a dashboard
 
-You can share a dashboard as a direct link or as a snapshot. You can also export a dashboard.
+You can share dashboards in the following ways:
 
-{{< admonition type="note" >}}
-If you change a dashboard, ensure that you save the changes before sharing.
-{{< /admonition >}}
+- Internally with a link
+- Externally with a link (to anyone)
+- Externally by email (to specific people)
+- As a snapshot
+- As a report
+- As a JSON file
+
+When you share a dashboard externally as a link or by email, those dashboards are included a list of your public dashboards. To view the list and manage these dashboards, navigate to **Dashboards > Public dashboards**.
+
+<!-- image of list here -->
+
+### Share internally
+
+Share a personalized, direct link to your dashboard within your organization.
 
 1. Click **Dashboards** in the main menu.
 1. Click the dashboard you want to share.
-1. Click **Share** in the top-right corner.
-
-   The share dialog opens and shows the Link tab.
+1. Click the **Share** drop-down in the top-right corner and select **Share internally**.
+1. (Optional) Set the following options (they're enabled by default):
+   - **Lock time range** - Change the current relative time range to an absolute time range.
+   - **Shorten link** - Shorten the dashboard link.
+1. Select the **Current**, **Dark**, or **Light** theme for the dashboard.
+1. Click **Copy link**.
+1. Send the copied link to a Grafana user with authorization to view the link.
+1. Click the **X** at the top-right corner to close the share drawer.
 
-### Share a direct link
+### Share externally with specific people
 
-The **Link** tab shows the current time range, template variables, and the default theme. You can also share a shortened URL.
+To share with specific external users, you can send them a link by email. Use this option when you only want to share your dashboard with specific people instead of anyone who navigates to the link. Sharing a link by email creates a 30-day token associated with that external user.
```

Comment:
```suggestion
To share with specific external users, you can send them a link by email. Use this option when you only want to share your dashboard with specific people instead of anyone who navigates to the link. Sharing a link by email creates a 30-day token.
```